capty.py

```
@@ -7,15 +7,252 @@
 import os
 import signal
 import time
+import logging
```

Ruff (0.12.2)

Module level import not at top of file

(E402)